### DISCUSSION TITLE
Cannot see the powerpoint

### DISCUSSION THREAD

**Discussion by ignatandrei** [microsoft/community-content#3] [Season of AI]
My MVP profile is https://mvp.microsoft.com/en-US/MVP/profile/f95c80b4-3c9a-e411-93f2-9cb65495d3c4
My email is ignatandrei@yahoo.com

I cannot visualize  "Generative AI for dotnet Devs May2024.pptx" because

![image](https://github.com/microsoft/community-content/assets/153982/47505eb1-54ff-4746-9701-7fa605294444)

And when I try https://aka.ms/confidentialextended it says that it does not work 

Please help!

**Comment by nielsberglund** [comment]
Hi @ignatandrei!

Hmm, I don't have that issue. Do you have an M365 subscription (I'm using my MVP sub)?

**Reply by ignatandrei** [reply]
Yes, I do.

**Reply by Tbush-dev** [reply]
Apologies for the inconvenience! We've updated the permissions on that deck and re-uploaded the assets yesterday 6-May. Give it another try and let us know if you still face any issues.

**Reply by ignatandrei** [reply]
I can open the presentation now, thanks!
